package.json

```diff
@@ -27,30 +34,24 @@
   },
   "homepage": "https://github.com/gnosis/dapp-ui#readme",
   "scripts": {
-    "postinstall": "cd ./examples/ts-frontend && npm i && cd ../../",
     "start": "NODE_ENV=development webpack-dev-server --mode development --hot",
     "start:example-ts": "cd ./examples/ts-frontend && npm start && cd ../../",
-    "build": "rimraf ./dist && NODE_ENV=production webpack --mode production",
+    "build": "rimraf ./lib ./lib-esm && npm run build:commonjs && npm run build:esm",
+    "build:commonjs": "tsc",
+    "build:esm": "tsc -m es6 -t esnext --outDir lib-esm",
     "eslint": "eslint . --ext .ts,.tsx,.js",
     "eslint-fix": "eslint --fix . --ext .ts,.tsx,.js",
     "stats": "NODE_ENV=production webpack --mode production --env.stats --profile --json > stats.json && webpack-bundle-analyzer ./stats.json ./dist",
     "test": "jest --detectOpenHandles"
   },
   "dependencies": {
-    "@walletconnect/web3-provider": "^1.0.0-beta.32",
+    "@walletconnect/web3-provider": "^1.0.0-beta.34",
```

Comment:
I think this is not part of the scope of the PR

Reply:
I updated an already included library, to see if nothing breaks. Why? Because `^1.0.0-beta.32` covers the latest available -- `1.0.0-beta.34`, which will get installed during dependencies install with `npm i gnosis/dapp-ui.git`
You can check here https://semver.npmjs.com/